### PR TITLE
AMP caption icon and a little bit of Sass

### PR DESCRIPTION
**Pull request by zeftilldeath**
## What does this change?
Oh AMP you gettin' SASSY.

## What is the value of this and can you measure success?
Easier for future devs, from the future. 

## Does this affect other platforms - Amp, Apps, etc?
screen shot 2016-12-06 at 16 42 06

## Screenshots
screen shot 2016-12-06 at 16 42 25

## Request for comment
@NataliaLKB

### CODE REVIEW

File: static/src/stylesheets/amp/_inlineSvgs.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    text-align: center;
+}
+
+.fc-item {
+    .inline-quote svg {
+        width: 19px;
+        height: 12px;
+    }
+
+    .fc-item .inline-video-icon svg,
```
Comment (NataliaLKB) [inline static/src/stylesheets/amp/_inlineSvgs.scss]: We can now use `.inline-video-icon__svg` 😄

File: static/src/stylesheets/amp/_social.scss [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.submeta hr, .meta__extras {
+    border: 0;
+    border-top: 1px dotted $neutral-4;
+}
+
+.social-icon {
+    &--facebook {
+        background-color: #3067a3;
```
Comment (NataliaLKB) [inline static/src/stylesheets/amp/_social.scss]: Do we have variables for this?

**Comment by NataliaLKB** [comment]
a couple questions, but otherwise great!

File: static/src/stylesheets/amp/_social.scss (hunk 2 above)
Comment (zeftilldeath) [inline static/src/stylesheets/amp/_social.scss]: Good spot yes we do!

**Review by NataliaLKB — APPROVED** [review]
👍

**Pull request merged**